build/Gruntfile.js

```diff
@@ -31,9 +31,10 @@ module.exports = function(grunt) {
             beforeJS: true,
             dest: '<%= DIR_BASE %>/concrete/js/build/core/image-editor/image-editor.js',
             src: [
+                '<%= DIR_BASE %>/concrete/js/build/core/image-editor/build/_start.js',
                 '<%= DIR_BASE %>/concrete/js/build/core/image-editor/build/kinetic.prototype.js',
                 '<%= DIR_BASE %>/concrete/js/build/core/image-editor/build/imageeditor.js',
-                '<%= DIR_BASE %>/concrete/js/build/core/image-editor/build/history.js.js',
+                //'<%= DIR_BASE %>/concrete/js/build/core/image-editor/build/history.js',
```

Comment:
@KorvinSzanto I don't know the ImageEditor that much, but we corrently don't include `image-editor/build/history.js` in it at the moment (because the filename is mispelled - `history.js.js` instead of `history.js`).
So, for now I just commented out this line.
What should we do with that file?